etc/omise_error_mapping.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<mapping xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Payment:etc/error_mapping.xsd">
+    <message_list>
```

Comment:
Do we need from any UI perspective to display so detailed info to user?
like `Card was marked as fraudulent`

Reply:
I agree that we do not need to display a very detailed error message to the user.
However, I do not think that it is correct that all errors return a single message as in the current implementation.
What do you think of using i18n to replace messages for users?

Reply:
I would think if if a user's card had been marked as fraudulent, they would want to know about it! I certainly would

Reply:
If error messages being shown to the user are not currently translated to the store's current language, I think they should be

Reply:
@jonrandy I will add the translation in the next PR.

Reply:
It would make more sense to add it to this PR really. It is directly related